weave/weave_types.py

```diff
@@ -1075,6 +1092,12 @@ def load_instance(self, artifact, name, extra=None):
         mapper = mappers_python.map_from_python(self, artifact)
         return mapper.apply(result)
 
+    def __eq__(self, other) -> bool:
```

Comment:
Hmm, why did you need this? I thought we had implemented __hash__ everywhere which made all types comparable with __eq__ but maybe I'm wrong about that.

Reply:
It's because all `ObjectType`s are currently equal in weave,

```
In [2]: from weave import weave_types as types

In [3]: ot1 = types.ObjectType(a=types.Int())

In [4]: ot2 = types.ObjectType(b=types.String())

In [5]: ot1 == ot2
Out[5]: True
```

on master.